sdk-extensions/autoconfigure/build.gradle.kts

```diff
@@ -117,22 +122,30 @@ tasks {
     jvmArgs("-Dotel.java.disabled.resource-providers=io.opentelemetry.sdk.extension.resources.OsResourceProvider,io.opentelemetry.sdk.extension.resources.ProcessResourceProvider")
     // Properties win, this is ignored.
     environment("OTEL_JAVA_DISABLED_RESOURCE_PROVIDERS", "io.opentelemetry.sdk.extension.resources.ProcessRuntimeResourceProvider")
+    environment("OTEL_TRACES_EXPORTER", "none")
+    environment("OTEL_METRICS_EXPORTER", "none")
   }
 
   val testResourceDisabledByEnv by existing(Test::class) {
     environment("OTEL_JAVA_DISABLED_RESOURCE_PROVIDERS", "io.opentelemetry.sdk.extension.resources.OsResourceProvider,io.opentelemetry.sdk.extension.resources.ProcessResourceProvider")
+    environment("OTEL_TRACES_EXPORTER", "none")
+    environment("OTEL_METRICS_EXPORTER", "none")
   }
 
+  val testResourceNotInitialized by existing(Test::class)
+
   val check by existing {
     dependsOn(
       testConfigError,
       testFullConfig,
+      testInitializeRegistersGlobal,
```

Comment:
Turns out those tests weren't even added to the `check` task - I've added and fixed them, they were pretty outdated